src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  font-family: "Roboto", sans-serif;
```

Comment:
Checklist:
2. [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/c0b55616-cfe6-46b1-a854-dafe7005e892)

Also, as it's the main font it's better to apply it for the html or body, let's moe it.